src/trip.core.js

```diff
@@ -479,7 +479,7 @@ Trip.prototype = {
    * @param {Object} o
    */
   showCurrentTrip: function(o) {
-    if (this.settings.enableAnimation) {
+    if (TripUtils.isSet(o.enableAnimation, this.settings.enableAnimation)) {
```

Comment:
@71FIL if we are going to add this to global, then this `if` should be removed and call `this.removeAnimation()` directly every time.

For example, if you set global `enableAnimation` to `false`, and the first trip we set the local one to `true`, and the second trip we set it back to `false`.

In this case, for the first trip, it's right, so we'll add the animation classes to `trip-block` to make sure it can be animated. But when the first trip ends and jumps to the second trip, because its local `enableAnimation` is set to `false` and the global `enableAnimation` is also `false`, there is no way to jump into this block to remove previous animation, so it will still be animated !